docs/docs/guide/2_api-reference/ui/settingsItem.md

```diff
@@ -0,0 +1,52 @@
+# Settings Item
+
+### Add settings item
+Add a new settings item to the Shopware settings. The content of the settings item module is determined by your `locationId`. 
+A specific view or a set of actions can be triggered based on the `locationId`.
+
+#### Usage:  
+```ts
+ui.settings.addSettingsItem({
+    label: 'App Settings',
+    locationId: 'settings-location-id',
+    icon: icons.defaultObjectBriefcase,
+    displaySearchBar: true,
+    tab: 'plugins',
+});
+```
+
+#### Parameters
+| Name                 | Required | Default        | Description                                                   |
+| :------------------- | :------- | :------------- | :------------------------------------------------------------ |
+| `label`              | true     |                | The label of the tab bar item                                 |
+| `locationId`         | true     |                | The Id for for the component section in the tab content       |
+| `icon`               | true     |                | The icon to display in your settings item                     |
+| `displaySearchBar`   | false    | true           | Toggles the sw-page search bar on/off                         |
+| `tab`                | false    | 'plugins'      | Determines in which tab your settings item will be displayed  |
+
+### Getting the right icon
+A list of valid icon names is exported by the extension sdk, see below example on how ot get them.
+```ts
+import { icons } from '@shopware-ag/admin-extension-sdk';
```

Comment:
This would import all icons from the icon list. This leads to a larger bundle size then necessary. I would prefer a direct import aka:

```js
import { defaultObjectBriefcase } from '@shopware-ag/admin-extension-sdk/es/icons';
```

This would be tree-shakable

Reply:
Understood 👍

Reply:
Or better: replace the object with just a type. Then has the user autocompletion and don't need to import anything which increases the bundle size: https://github.com/shopware/admin-extension-sdk/pull/8/files#r784908204